--- configure.orig	2016-04-21 07:55:50.000000000 -0700
+++ configure	2016-05-11 10:47:53.000000000 -0700
@@ -5449,9 +5449,9 @@
 $as_echo "$as_me: serf library configuration via prefix" >&6;}
   serf_required=yes
   for serf_major in serf-2 serf-1; do
-    if ! test -d $serf_prefix/include/$serf_major; then continue; fi
+    if ! test -d $serf_prefix/local/include/$serf_major; then continue; fi
     save_cppflags="$CPPFLAGS"
-    CPPFLAGS="$CPPFLAGS $SVN_APR_INCLUDES $SVN_APRUTIL_INCLUDES -I$serf_prefix/include/$serf_major"
+    CPPFLAGS="$CPPFLAGS $SVN_APR_INCLUDES $SVN_APRUTIL_INCLUDES -I$serf_prefix/local/include/$serf_major"
     for ac_header in serf.h
 do :
   ac_fn_c_check_header_mongrel "$LINENO" "serf.h" "ac_cv_header_serf_h" "$ac_includes_default"
@@ -5559,7 +5559,7 @@
   done
 
   if test $serf_found = "yes"; then
-    SVN_SERF_INCLUDES="-I$serf_prefix/include/$serf_major"
+    SVN_SERF_INCLUDES="-I$serf_prefix/local/include/$serf_major"
     if test -e "$serf_prefix/lib/lib$serf_major.la"; then
       SVN_SERF_LIBS="$serf_prefix/lib/lib$serf_major.la"
     else
